unit-tests/algo/convertions/test-y411.cpp

```diff
@@ -0,0 +1,142 @@
+// License: Apache 2.0. See LICENSE file in root directory.
```

Comment:
conversions/ is new, yes?
Why not proc/, to fit the directory name in src/proc/?